executor/executor.go

```diff
@@ -763,9 +763,9 @@ func (e *HashJoinExec) runJoinWorker(idx int) {
 	e.wg.Done()
 }
 
-// joinOneBigRow creates result rows from a big table Row and send them to resultRows channel.
-// Every matched row generates a result row.
-// Ff there is no matched rows and it is outer join, a null filled result row is created.
+// joinOneBigRow creates result rows from a row in a big table and sends them to resultRows channel.
+// Every matching row generates a result row.
+// If there is no matching rows and it is outer join, a null filled result row is created.
```

Comment:
there is -> there are
a null filled result row is created. -> a result row whose value is NULL  is created.